### PR TITLE
fix: Description and min/max characters not visible during error screen

### PR DESCRIPTION
**Pull request by gitstart**
Fixes https://github.com/strapi/strapi/issues/16290

### What does it do?
- Unclear max/min error messages with incorrect input length

### Why is it needed?
- Visibility of the minimum / maximum characters and description during the error.

### How to test it?
1. Add a collection type with a text field and set minimum / maximum characters and add a description.
2. Add content within this new collection type
3. Notice the minimum/maximum characters and description below the field.
4. Add less or more characters in the field.
5. Publish the content
6. The error message is descriptive enough to correct that error.

## Demo
Screenshot 2023-04-14 at 6 40 27 PM

### CODE REVIEW

**Comment by gitstart** [comment]
Hi @joshuaellis This PR is ready for review, kindly take a look.

File: packages/core/admin/admin/src/translations/en.json [hunk 1]
```diff
@@ -753,7 +753,7 @@
   "content-manager.form.Input.filters": "Enable filters",
   "content-manager.form.Input.hint.character.unit": "{maxValue, plural, one { character} other { characters}}",
   "content-manager.form.Input.hint.minMaxDivider": " / ",
-  "content-manager.form.Input.hint.text": "{min, select, undefined {} other {min. {min}}}{divider}{max, select, undefined {} other {max. {max}}}{unit}{br}{description}",
+  "content-manager.form.Input.hint.text": "{min, select, undefined {} other {min. (min: {min})}}{divider}{max, select, undefined {} other {max. (max: {max})}}{unit}{br}{description}",
```
Comment (jhoward1994) [inline packages/core/admin/admin/src/translations/en.json:756]: Thanks! The other translations looks good.

I think this was better before though as I find "min. (min: 5) / max. (max: 100) characters" (e.g.) a bit strange to read

Comment (raph941) [reply packages/core/admin/admin/src/translations/en.json]: Thanks @jhoward1994,
This translation has now been reverted to previous state.

Comment (gitstart) [reply packages/core/admin/admin/src/translations/en.json]: Thanks @jhoward1994,
This translation has now been reverted to previous state.

**Review by jhoward1994 — APPROVED** [review]
Thank you!

**Pull request merged**